src/EventRegistry.php

```diff
@@ -52,11 +59,30 @@ class EventRegistry
         'security.switch_user' => SwitchUserEvent::class,
     ];
 
+    private static $newEventsMap = [
+        'kernel.controller_arguments' => ControllerArgumentsEvent::class,
+        'kernel.controller' => ControllerEvent::class,
+        'kernel.response' => ResponseEvent::class,
+        'kernel.request' => RequestEvent::class,
+        'kernel.view' => ViewEvent::class,
+        'kernel.exception' => ExceptionEvent::class,
+        'kernel.terminate' => TerminateEvent::class,
+    ];
+
     private $eventDispatcher;
 
     public function __construct(EventDispatcherInterface $eventDispatcher)
     {
         $this->eventDispatcher = $eventDispatcher;
+
+        // Loop through the new event classes
+        foreach (self::$newEventsMap as $oldEventName => $newEventClass) {
+            //Check if the new event classes exist, if so replace the old one with the new.
+            if (class_exists($newEventClass)) {
+                unset(self::$eventsMap[$oldEventName]);
+                self::$eventsMap[$newEventClass] = $newEventClass;
+            }
+        }
```

Comment:
Let's add a test for this :). Actually, we have one: `EventRegistryTest::testGetEventClassNameFromStandardList()`. But now, I think it needs to have a different "assert" value based on whether or not the `ExceptionEvent` class existss.

Reply:
I changed this bit in order to display just the Classname and not the FQCN. This fixes the first part of the bug we encounter in the subscriber.tpl.php and displays the list of events in the CLI much better.

Reply:
Did you push those changes? Or are you saying you already did something previously, and what I was saying aren't issues?